src/Tribe/Main.php

```diff
@@ -33,7 +33,7 @@ class Tribe__Events__Main {
 		const VENUE_POST_TYPE     = 'tribe_venue';
 		const ORGANIZER_POST_TYPE = 'tribe_organizer';
 
-		const VERSION             = '5.1.1';
+		const VERSION             = '5.1.2';
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 13 spaces

<sub>hash: 1eb125f5345e78d2763bd5b0802a3232</sub>